### DISCUSSION TITLE
YouTube logo top lefthand corner

### DISCUSSION THREAD

**Discussion by talhaazam** [ReVanced/revanced-discussions#1083] [Question] [answered]
Hi everyone,
I have downloaded a version of YouTube Revanced extended 
How ever the logo on top left hand corner is not YouTube original one.
Can i change the logo through editing the apk or something?
I have tried to edit the APK but could not find where this logo is comming from.
Any help would be greatly appreciated..
![Screenshot_20241027-125006](https://github.com/user-attachments/assets/6dc41a30-ac87-461c-a7ce-0f1e14f8496c)

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
You're not using the official ReVanced, and you probably downloaded a pre-patched apk so who knows what you installed.

Uninstall that, change your google password just to be safe, and install using ReVanced manager available at: [revanced.app](http://revanced.app)
